test/runtests.jl

```diff
@@ -341,7 +341,7 @@ end
 let weakdict = pyimport("weakref")["WeakValueDictionary"]
     # (use weakdict for the value, since Python supports
     #  weak references to type objects)
-    @test weakdict(Dict(3=>weakdict)) == Dict(3=>weakdict)
+    @test weakdict(Dict(3=>weakdict))[:__getitem__](3) == weakdict
```

Comment:
Hmm, actually it calls `PySequence_GetItem` which is not quite the same thing.  It should actually be changed to call `PyObject_GetItem`, which does the right thing (calling `__getitem__` for a mapping and `PySequence_GetItem` for a sequence).